cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -253,7 +254,7 @@ private Chapter buildChapterForPublicationContext() {
         } else if (isJournal(cristinObject) && isJournalCorrigendum(cristinObject)) {
             return createJournalCorrigendum();
         } else if (isJournal(cristinObject) && isJournalArticle(cristinObject)) {
-            return createJournalArticle();
+            return createJournalArticle(cristinObject.getSecondaryCategory());
```

Comment:
cristinObject is a field in the class. You do not need to pass it as a parameter